packages/e2e/README.md

```diff
@@ -107,6 +107,18 @@ For our e2e tests, we use a specific pattern for test user creation in our vario
 
 3. Enter their password as `.env.VALID_PASSWORD` and create the user
 
+#### Creating an FORCE_CHANGE_PASSWORD user
```

Comment:
This was missing from existing README (required when you set up new environment)